lib/Fez/Util/Config.rakumod

```diff
@@ -2,16 +2,48 @@ unit module Fez::Util::Config;
 
 use Fez::Util::Json;
 
-state $CONFIG-PATH = (%*ENV<FEZ_CONFIG>//%?RESOURCES<config.json>).IO;
-state $CONFIG      = from-j($CONFIG-PATH.slurp);
+state $FALLBACK-CONFIG-PATH = %?RESOURCES<config.json>.IO;
```

Comment:
I don't like not being able to override where the config is with an ENV var - this is useful for things likes cron scripts.